services/kuber/server/server.go

```diff
@@ -100,6 +96,18 @@ func (s *server) StoreClusterMetadata(ctx context.Context, req *pb.StoreClusterM
 		return nil, fmt.Errorf("failed to marshal %s cluster metadata: %w", req.GetCluster().GetClusterInfo().GetName(), err)
 	}
 
+	// local deployment - print metadata
+	if namespace := envs.Namespace; namespace == "" {
+		// NOTE: DEBUG print
+		// var buffer bytes.Buffer
+		// for node, ips := range md.NodeIps {
+		// 	buffer.WriteString(fmt.Sprintf("%s: %v \t| %v \n", node, ips.PublicIP, ips.PrivateIP))
+		// }
+		// buffer.WriteString(fmt.Sprintf("%s\n", md.PrivateKey))
+		// log.Info().Msgf("Cluster metadata from cluster %s \n%s", req.GetCluster().ClusterInfo.Name, buffer.String())
```

Comment:
I would uncomment this and just use log.Debug()

Reply:
I was considering it, however, I am not sure how I feel about it. I would rather not have sensitive information in container logs.